bench/README.md

```diff
@@ -18,40 +28,214 @@ pip3 install -r bench/requirements.txt
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine. `Number of users should be equal to number of workers for write tests.`
+
+
+## Web UI
+
+To run write load tests
+
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py WriteModelLoadTest --master -H http://master:8089 
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: -f /dcl/bench/locustfile.py WriteModelLoadTest --worker --master-host master
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine. `Number of users should be equal to number of workers for write tests.`
+
+To run read load tests
+
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py ReadModelLoadTest --master -H http://master:8089 
+    ```
+    e.g.: for `worker`: 
+    ```yml
+    command: -f /dcl/bench/locustfile.py ReadModelLoadTest --worker --master-host master
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
+
+To run write/read load tests
+
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py --master -H http://master:8089 
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: -f /dcl/bench/locustfile.py --worker --master-host master
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine. `Number of users should be equal to number of workers for write tests.`
+
+Then you can open `http://localhost:8089/` and launch the tests from the browser.
+
+# For Local Running
```

Comment:
`Running tests against local pool` would be a better title for this section

Reply:
Done! ✅